.gitignore

```diff
@@ -11,16 +8,26 @@ loader/
 log.txt
 *.sublime-project
 *.sublime-workspace
+.DS_Store
+Thumbs.db
+UserInterfaceState.xcuserstate
+.env
 
-.stencil/
+# Manifold
+.manifold.yml
+
+# Stencil
+!www/favicon.ico
 .idea/
-.vscode/
 .sass-cache/
+.stencil/
 .versions/
+.vscode/
+coverage/
+dist/
 node_modules/
+pkg/
```

Comment:
Last thing we can do to kill the Node script is just to publish from `./` rather than `./pkg`.

That was mostly done to prevent managing the version in package.json, but we ended up doing it anyway. It was also done to prevent accidentally uploading the whole docs site to npm, but that can be prevented with a `.npmignore` file:

```
*
!./dist
```

You may have to `npm publish --dry-run` to make sure only the bare minimum makes it to npm.

Anyway, with all these suggestions I think we could kill that Node script and make things easier to manage. What do you think?

Reply:
>but we ended up doing it anyway

For more context, the reason that `package.json` has to change had to do with the `rollup-plugin-replace` plugin that replaced `<@UI_VERSION@>` with the version for `package.json`. In the environments it ran in, it didn’t always have access to the Git context, which meant it couldn’t listen to Git tags. It needed to pull from a reliable, hard-coded source.

So the UI scripts kinda evolved over time to meet shifting needs, and I’d redo it to be simpler if I had to start from scratch today.